src/components/foooter/Footer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FaInstagram, FaFacebook, FaTwitter } from 'react-icons/fa';
+
+function Footer() {
+  return (
+    <div className="footer bg-white h-auto flex flex-col sm:flex-col  justify-between px-8 py-2 pt-10 gap-5 sm:gap-2 max-w-full">
+      <div className="flex flex-col sm:flex-row sm:gap-20">
+        <div className="footer__links flex  justify-center sm:justify-start">
+          <a href="/about-us" className="mr-6">
+            About us
+          </a>
+          <a href="/help" className="mr-6">
+            Help & contact
+          </a>
+          <a href="/policies" className="mr-6">
+            Policies
+          </a>
+        </div>
+        <div className="footer__social flex justify-center items-center sm:ml-auto">
+          <span className="mr-4">Follow us:</span>
+          <a href="https://www.instagram.com/" className="mr-3">
+            <FaInstagram />
+          </a>
+          <a href="https://www.facebook.com/" className="mr-3">
+            <FaFacebook />
+          </a>
+          <a href="https://www.twitter.com/" className="mr-3">
+            <FaTwitter />
+          </a>
+        </div>
+      </div>
+      <div className="footer__copyight text-center">
+        <span className="text-gray-700 h-6"> &copy; Team Cypher, 2023. All rights reserved. </span>
```

Comment:
On this line, just like @shyakadavis suggested, i think it's best you get the year dynamically with some JS code rather than hard coding it to ```2023```

Reply:
done